pagefaultgames/pokerogue
# [Sprite] Fixed shinies for Zubat, Golett, female Garchomp

Converted to draft while awaiting further instruction from the Head Artists.
## What are the changes?

- Fixed shiny/rare Zubat, Golbat, Crobat
- Fixed inconsistencies in Garchomp's shiny
- Fixed Golett's shiny front sprite still using base replacement

## Why am I doing these changes?



- Zubat was meant to have its base replacement removed. This was not done correctly, resulting in its shiny and rare variant colors being mixed up, and also different between male and female.
- Shiny Garchomp was also very slightly different based on gender which was unintentional.
- Golurk was meant to have its base replacement removed a month ago
![image](https://github.com/pagefaultgames/pokerogue/assets/36677462/398ce8e3-6494-41ab-9fb2-ba19afbd97bd)
![image](https://github.com/pagefaultgames/pokerogue/assets/36677462/f80251c5-a6f8-4a76-86b2-0e12eff8a4a5)

## What did change?


- Zubat line entirely restored to original shiny color, male rare variants fixed
- Female Garchomp now matches male's colors
- Golett shiny front sprite fixed
- Removed entries in _masterlist.json for the genies and Palossand, whose base replacements were reverted (having their entries allowed their nonexistent variants to be caught)

### Screenshots/Videos


Before:
![image](https://github.com/pagefaultgames/pokerogue/assets/36677462/3321af4f-9162-4ec0-93f9-b5b9af1e2ebb)
<img width="776" alt="Screenshot_766" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/4ae00956-b6c4-4c7c-b4fd-598371ad4ab6">
<img width="776" alt="Screenshot_768" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/671dcc04-eb74-4bcf-84d6-0da280f0285c">
<img width="260" alt="Screenshot_770" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/8fa7ea43-44bb-45f7-9267-956b369423c0">
<img width="779" alt="Screenshot_771" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/3c7ad6b9-70c9-4fc2-bdce-ff5a75307c52">
<img width="251" alt="Screenshot_772" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/ce9ae499-f86f-4aef-a5de-2c42af451767">
<img width="769" alt="Screenshot_773" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/4be07f02-4a5c-45d6-a20d-0bdb873c416a">
<img width="249" alt="Screenshot_774" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/62d97929-4f3b-48f2-a09c-3e4c455595ef">
After:

<img width="775" alt="Screenshot_775" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/340732bf-e669-4e86-9518-e4db26739e1c">
<img width="778" alt="Screenshot_776" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/6859136c-0b8c-4e83-860e-b61c8d3c3257">
<img width="238" alt="Screenshot_777" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/fd8d8420-2a8a-454b-be34-7e63007fc3e4">
<img width="254" alt="Screenshot_778" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/bf5203d1-1b24-4b35-bf76-97e98a4c6a03">
<img width="771" alt="Screenshot_779" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/bb6cd665-194d-48d2-8783-d1c13ba23bc8">
<img width="777" alt="Screenshot_780" src="https://github.com/pagefaultgames/pokerogue/assets/36677462/f8fd119f-35df-4042-a444-1ad4fcef6548">

## How to test the changes?




- Use a save file with every variant and shiny.
- Check every affected mon in game to see if their shinies/variants are correctly colored.

## Checklist
